src/main/kotlin/racing/domain/Cars.kt

```diff
@@ -0,0 +1,26 @@
+package racing.domain
+
+private const val MINIMUM_COUNT_OF_CARS = 2
+
+class Cars(private val cars: List<Car>) {
+
+    init {
+        require(cars.size >= MINIMUM_COUNT_OF_CARS) {
+            "자동차는 최소 ${MINIMUM_COUNT_OF_CARS}대 이상이어야 합니다."
+        }
+    }
+
+    constructor(numberOfCars: Int) : this(
+        (1..numberOfCars).map { Car() }
+    )
+
+    fun move(movable: Movable) {
+        cars.forEach { car ->
+            car.move(movable.move())
+        }
```

Comment:
람다식의 람다 파라미터가 한개뿐이라면 **x -> x....** 표현 없이 바로 `it`으로 사용할 수 있어요! 😃 
```suggestion
        cars.forEach { it.move(movable.move()) }
```